src/main/java/org/prebid/server/settings/CachingApplicationSettings.java

```diff
@@ -23,6 +24,7 @@
     private final ApplicationSettings delegate;
 
     private final Map<String, Account> accountCache;
+    private final Map<String, String> accountToExceptionMessageCache;
```

Comment:
IMO this naming looks quite long and confusing, how about we rename it to something like `exceptionsByAccounts`? Let's see what @rpanchyk thinks.

Reply:
Agree, or smt like accountToErrorCache.